packages/Microsoft.Bot.Components.Telephony/Actions/CallTransferDialog.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using AdaptiveExpressions.Properties;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+using Newtonsoft.Json;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Components.Telephony.Actions
+{
+    public class CallTransferDialog : Dialog
+    {
+        [JsonConstructor]
+        public CallTransferDialog([CallerFilePath] string sourceFilePath = "", [CallerLineNumber] int sourceLineNumber = 0)
+            : base()
+        {
+            // enable instances of this command as debug break point
+            this.RegisterSourceLocation(sourceFilePath, sourceLineNumber);
+        }
+
+        [JsonProperty("$kind")]
+        public const string Kind = "CallTransferDialog";
+
+        [JsonProperty("targetPhoneNumber")]
+        public StringExpression TargetPhoneNumber { get; set; }
```

Comment:
Naming: Should we consider just `PhoneNumber`?

Reply:
sure I think that should be fine

Reply:
We can discuss this one with the team, leaving unresolved so we discuss before merging

Reply:
This has been updated already resolving now